benchmarks/torchbench/requirements-bench.txt

```diff
@@ -1,3 +1,4 @@
-torch
-torchvision
-torchtext
+torch==1.10.2+cu111
+torchvision==0.11.3+cu111
+torchtext==0.11.*
+--extra-index-url=https://download.pytorch.org/whl/cu111
```

Comment:
Should we force a torch / cuda versions here or in a container?

Reply:
Here would be good enough, I think.